pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -3340,11 +3339,13 @@ private AutoTopicCreationOverride getAutoTopicCreationOverride(final TopicName t
         return null;
     }
 
+    @Deprecated
```

Comment:
Deprecate method should be explained, and new method suggested

Reply:
+1

Reply:
added